src/main/scala/cromwell/binding/Call.scala

```diff
@@ -69,6 +72,7 @@ case class Call(alias: Option[String],
                 task: Task,
                 inputMappings: Map[String, WdlExpression]) extends Scope {
   val name: String = alias getOrElse taskFqn
+  val callID: CallID = CallID(UUID.randomUUID())
```

Comment:
I con't think you need to explicitly type this as the name of the val is `callID` and it's explicitly created with an apply to `CallID`

Also now that I notice it I'd prefer to not capitalize abbreviations unless we're forced to (i.e. coming in from someone else's code like `UUID`)